### ISSUE TITLE
Normalize CSS should address FF's use of text decoration on `abbr` tag

### ISSUE THREAD

**Issue by wesruv** [backdrop/backdrop-issues#2314]
Firefox seems to be using:
`text-decoration: 1px dotted;` (or something similar), no other browser does, should be normalized with Backdrop's core CSS.

**Comment by wesruv** [comment]
We might consider just updating to the latest version:
https://github.com/necolas/normalize.css/blob/master/normalize.css

**Comment by klonos** [comment]
Yeah, certain articles (like this one for example) suggest using `outline: none;` on the `:focus` pseudo-selector while some others suggest using the `:-moz-focusring` instead. I tried both and none of them seems to work.

As for normalize.css, the changelog mentions these two following items for 4.0.0 (we are using 3.0.1 currently):

> - Correct inconsistent `text-decoration` and `border-bottom` for `abbr[title]` in Chrome, Edge, Firefox IE, Opera, and Safari.
> ...
> - Update normalization of `outline` for `:-moz-focusring` removed by a previous normalization in Firefox.

...I tried replacing our 3.0.1 with 4.0.0 and then the latest 5.0.0, but unfortunately none of them solved the issue. Must be a Firefox bug, but still would be a good idea to update normalize.css and keep it updated with feature versions.

**Comment by klonos** [comment]
...and let me upload some screenshots so that everyone understands what we are talking about @wesruv (because I pm'ed you on Gitter).

**Comment by klonos** [comment]
This is what the "required" asterisks currently look like on our forms in Chrome:

![](https://files.gitter.im/klonos/7gaa/backdrop-required_asterisk_on_forms-chrome.png)

...and this is how they look in Firefox:

![](https://files.gitter.im/klonos/fMvS/backdrop-required_asterisk_on_forms-firefox.png)

...here's how they look respectively with the changes from https://github.com/backdrop/backdrop/pull/1629 applied:

![](https://files.gitter.im/klonos/zsrs/backdrop-required_asterisk_on_forms-chrome-dots_removed.png)

![](https://files.gitter.im/klonos/7fdb/backdrop-required_asterisk_on_forms-firefox-dots_not_removed.png)
